src/altimeter.rs

```diff
@@ -0,0 +1,273 @@
+use bmp388::{config::{FifoConfig, OversamplingConfig}, Blocking, PowerControl, BMP388};
+use heapless::Vec;
+use core::{cell::RefCell, ptr::addr_of_mut, sync::atomic::AtomicBool};
+use cortex_m::interrupt::Mutex;
+use stm32f4xx_hal::{
+    i2c::dma::{
+            I2CMasterHandleIT, I2CMasterWriteReadDMA, I2cCompleteCallback
+        }, interrupt
+};
+use stm32f4xx_hal::i2c;
+use stm32f4xx_hal::timer::delay::SysDelay;
+use crate::futures::YieldFuture;
+use crate::pins::Altimeter;
+use crate::bmp581::I2c1Handle;
+use crate::bmp581::BMP581;
+
+const BMP388_FRAME_COUNT: usize = 73;
+const BMP388_BUF_SIZE:    usize = 512;
+const BMP581_FRAME_COUNT: usize = 16;
+const BMP581_BUF_SIZE:    usize = BMP581_FRAME_COUNT * 6;
```

Comment:
Could you make these associated constants on `BMP581` and `BMP388Wrapper`

Reply:
Fixed